src/rating/rating.ts

```diff
@@ -210,7 +210,7 @@ export class NgbRating implements ControlValueAccessor,
       return 100;
     }
     if (diff < 1 && diff > 0) {
-      return Number.parseInt((diff * 100).toFixed(2));
+      return Number.parseInt((diff * 100).toFixed(2), 10);
```

Comment:
Saw your comment, but I absolutely don't see the link to this PR and this file didn't have any issues previously. Could you please remove it?